sync_diff_inspector/chunk/chunk.go

```diff
@@ -0,0 +1,460 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package chunk
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	"github.com/pingcap/tidb-tools/pkg/dbutil"
+	"go.uber.org/zap"
+)
+
+const (
+	lt  = "<"
+	lte = "<="
+	gt  = ">"
+)
+
+type ChunkType int
+
+const (
+	Bucket ChunkType = iota + 1
+	Random
+	Limit
+	Others
+	Empty
+)
+
+// Bound represents a bound for a column
+type Bound struct {
+	Column string `json:"column"`
+	Lower  string `json:"lower"`
```

Comment:
not sure if you have converted numeric type to string, so `11` will less than `2`